tests/test_tasks/test_classification_task.py

```diff
@@ -34,4 +26,14 @@ def test_download_task(self):
 
     def test_class_labels(self):
         task = get_task(self.task_id)
-        assert task.class_labels == ["tested_negative", "tested_positive"]
\ No newline at end of file
+        assert task.class_labels == ["tested_negative", "tested_positive"]
+
+
+def test_get_X_and_Y():
+    task = get_task(119)
+    X, Y = task.get_X_and_y()
+    assert X.shape == (768, 8)
+    assert isinstance(X, pd.DataFrame)
+    assert Y.shape == (768,)
+    assert isinstance(Y, pd.Series)
+    assert pd.api.types.is_categorical_dtype(Y)
```

Comment:
Has not much to do with the PR per se, was an artifact of experimenting with how/when fixtures are applied to `unittest` tests. Figured might as well leave it, one less test to move to pytest